tests/report_tests/plugin_tests/systemd.py

```diff
@@ -0,0 +1,92 @@
+# This file is part of the sos project: https://github.com/sosreport/sos
+#
+# This copyrighted material is made available to anyone wishing to use,
+# modify, copy, or redistribute it subject to the terms and conditions of
+# version 2 of the GNU General Public License.
+#
+# See the LICENSE file in the source distribution for further information.
+
+from sos_tests import StageOneReportTest, StageTwoReportTest
+
+
+class SystemdPluginTest(StageOneReportTest):
+    """Basic sanity check to make sure common config files are collected
+
+    :avocado: tags=stageone
+    """
+
+    sos_cmd = '-o systemd'
+
+    def test_systemd_files_collected(self):
+        self.assertFileCollected("/etc/systemd")
+        self.assertFileCollected("/lib/systemd/system")
+        self.assertFileCollected("/lib/systemd/user")
+        self.assertFileCollected("/etc/vconsole.conf")
+        self.assertFileCollected("/run/systemd/generator*")
```

Comment:
Hm.. sadly `assertFileCollected` does not work for globs, it just executes:

        if fname.startswith(('sos_', '/sos_')) or os.path.exists(fname):
            self.assertFileExists(self.get_name_in_archive(fname))
        else:
            assert True

And while having e.g. `/run/systemd/generator` directory on my system, `os.path.exists("/run/systemd/generator*")` returns False - so no test is executed if either `/run/systemd/generator*` stuff is collected.

Worth expanding the method to cover globs?

Reply:
+1 to expand the method to cover globs